forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -799,7 +799,7 @@ public final int staticReplaceDamage(final int damage, final Card source, final
                     restDamage = 2;
                 }
             } else if (c.getName().equals("Elderscale Wurm")) {
-                if (c.getController().equals(this) && getLife() - restDamage < 7) {
+                if (c.getController().equals(this) && getLife() >= 7) {
                     restDamage = getLife() - 7;
```

Comment:
this change doesn't make sense:
if life is 20 and 10 damage would be dealt the prediction would now be raised to 13

Reply:
Alrighty, tried to improve this prediction, though I admit that I suck at math, so I dunno :D